marklogic/rows.py

```diff
@@ -67,6 +67,61 @@ def query(
         not 2xx, then the entire response is always returned.
         """
         path = "v1/rows/graphql" if graphql else "v1/rows"
+        return self.__send_request(
+            path,
+            dsl,
+            plan,
+            sql,
+            sparql,
+            graphql,
+            format,
+            tx,
+            return_response,
+            **kwargs,
+        )
+
+    def update(
+        self,
+        dsl: str = None,
+        format: str = "json",
+        tx: Transaction = None,
+        return_response: bool = False,
+        **kwargs,
+    ):
+        """
+        Sends an update query to an endpoint at the MarkLogic rows service defined at
+        https://docs.marklogic.com/REST/client/row-management. Note that this feature
+        requires the use of MarkLogic version 11.2 or later.
+
+        For more information about Optic Update and using the Optic DSL,
+        see https://docs.marklogic.com/guide/app-dev/OpticAPI.
+        TODO - add links for Optic Update.
+
+        :param dsl: an Optic DSL query
+        :param tx: optional REST transaction in which to service this request.
+        :param return_response: boolean specifying if the entire original response
+        object should be returned (True) or if only the data should be returned (False)
+        upon a success (2xx) response. Note that if the status code of the response is
+        not 2xx, then the entire response is always returned.
+        """
+        path = "v1/rows/update"
+        return self.__send_request(
+            path, dsl, None, None, None, False, format, tx, return_response, **kwargs
```

Comment:
One tweak - I don't think `False` is correct, though I don't think you need to pass in any of the args that default to None that you'll never have values for. I think this works (could be wrong):

    ("v1/rows/update", dsl, format=format, tx=tx, return_response=return_response, **kwargs)

Actually, a better approach would be to put the common args first in `send_request` (so move up format/tx/return_response), and then I don't think you need named args, this should work as well:

    ("v1/rows/update", dsl, format, tx, return_response, **kwargs)

Just not sure if "kwargs" will get unpacked correctly there. Might need `kwargs=**kwargs` or possibly `kwargs=kwargs`.

Reply:
Forgot I said this - I think just change `False` to `None` and then merge away. That way we don't have to worry about what happens with `kwargs`.